service/src/main.rs

```diff
@@ -108,1023 +38,15 @@ mod utils;
 mod worker;
 mod worker_peers_updater;
 
-const VERSION: &str = env!("CARGO_PKG_VERSION");
-
-pub type EnclaveWorker =
-	Worker<Config, NodeApiFactory, Enclave, InitializationHandler<WorkerModeProvider>>;
-pub type Event = substrate_api_client::ac_node_api::EventRecord<RuntimeEvent, Hash>;
+#[cfg(feature = "link-binary")]
+mod main_impl;
```

Comment:
Extracted the whole file into the `main_impl`, mainly because I wanted to feature gate all its imports, but I think it actually gives a better overview of the crate in general, as the main.rs mostly lists the modules and executes the main functions now.